docker-for-mac/multi-arch.md

```diff
@@ -54,3 +54,134 @@ Notice that this time, the `uname -a` output shows `armv7l` and
 
 Multi-architecture support makes it easy to build <a href="https://blog.docker.com/2017/11/multi-arch-all-the-things/" target="_blank">multi-architecture Docker images</a> or experiment with ARM images and binaries from your Mac.
 
+## Buildx (Experimental)
+
+With the Docker Desktop 2.0.4.0 Edge release, Docker is making it easier than ever to develop containers on, and for Arm servers and devices. Using the standard Docker tooling and processes, you can start to build, push, pull, and run images seamlessly on different compute architectures. Note that you don't have to make any changes to Dockerfiles or source code to start building for Arm.
+
+Docker Desktop 2.0.4.0 Edge release introduces a new CLI command called `buildx`.  Buildx allows you to locally build multi-arch images, link them together with a manifest file, and push them all to a registry using a single command.  With the included emulation, you can transparently build more than just native images.  Buildx accomplishes this by adding new builder instances based on BuildKit, and leveraging Docker Desktop's technology stack to run non-native binaries.
```

Comment:
> Buildx allows you to locally build multi-arch images
not just locally. However, the included emulation (aka "leveraging Docker Desktop's tech stack", same thing) is only relevant for the local case. The remote case is for building multi-arch images seamlessly across nodes of different architectures. In both cases it makes building multi-arch images seamless.

Reply:
Updated the content